docs/markdown/Contributions/releases/release-process.md

```diff
@@ -70,23 +70,15 @@ If there are things that must be removed, you can either:
 2. Remove it yourself, either in the release prep or as a precursor PR.
 3. Bump the removal date back by one dev release.
 
-### 0c. Release candidates - cherry-pick relevant changes
+### 0c. Release candidates - Check for cherry-picks
```

Comment:
These docs are nice, but I wonder if it's a bit unclear what action(s) are expected of the releaser. Maybe it could be something like:

```markdown
# 0c. Release candidates - Check for cherry-picks

There's many instances of landing a change in `main` and then wanting it to also apply to older releases. This is indicated by the `needs-cherrypick` label on a pull request, and there's automation that attempts to automatically cherry-pick those changes back to the relevant branches.

This automation may not always succeed, so [check for any pending cherry-picks](https://github.com/pantsbuild/pants/pulls?q=is%3Apr+label%3Aneeds-cherrypick+is%3Aclosed), and find the relevant ones by looking at the milestone: for instance, if doing a release for 2.16, the relevant cherry-picks are those for milestone `2.16.x` or earlier.

The process may fail in two ways:

- The cherry-picking process failed, and tagged the PR with `auto-cherry-picking-failed`: follow the instructions in the comment on the pull request. (This likely means there's merge conflicts that require manual resolution.)
- the cherry-pick didn't (yet) run: trigger the automation manually by going to [the GitHub Action](https://github.com/pantsbuild/pants/actions/workflows/auto-cherry-picker.yaml), clicking on the "Run workflow" button, and providing the PR number.
```